### PR TITLE
Domain Update: OOni Run Activity

### PR DESCRIPTION
**Pull request by Filmaluco**

### CODE REVIEW

File: app/src/main/java/org/openobservatory/ooniprobe/activity/OoniRunActivity.java [hunk 1]
```diff
@@ -110,78 +96,71 @@ else if (Intent.ACTION_VIEW.equals(intent.getAction())) {
...
+				loadOutOfDate();
 			}
 		} else {
-			title.setText(R.string.OONIRun_InvalidParameter);
-			desc.setText(R.string.OONIRun_InvalidParameter_Msg);
-			run.setText(R.string.OONIRun_Close);
-			icon.setImageResource(R.drawable.question_mark);
-			iconBig.setImageResource(R.drawable.question_mark);
+			loadInvalidAttributes();
+		}
```
Comment (sdsantos) [inline app/src/main/java/org/openobservatory/ooniprobe/activity/OoniRunActivity.java:119]: What a nice christmas tree 🎄

File: app/src/test/java/org/openobservatory/ooniprobe/domain/GetTestSuiteTest.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        attribute.urls.add(faker.internet.url());
+        attribute.urls.add(faker.internet.url());
+
+        // Act
+        AbstractSuite suite = getSuite.get(WebConnectivity.NAME, attribute);
+        AbstractTest[] tests = suite.getTestList(pm);
+
+        // Assert
+        Assert.assertEquals(WebsitesSuite.NAME, suite.getName());
+        Assert.assertEquals(1, tests.length);
```
Comment (sdsantos) [inline app/src/test/java/org/openobservatory/ooniprobe/domain/GetTestSuiteTest.java:42]: `assertArrayEquals`

File: app/src/test/java/org/openobservatory/ooniprobe/domain/VersionCompareTest.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+        // Act / Assert
+        Assert.assertEquals(-1, comparator.compare("1.0.0", "1.2.0"));
+        Assert.assertEquals(-1, comparator.compare("0.1.0", "0.1.2"));
+        Assert.assertEquals(-1, comparator.compare("0.0.1", "0.0.3"));
+    }
+
+    public VersionCompare build() {
+        return new VersionCompare();
+    }
```
Comment (sdsantos) [inline app/src/test/java/org/openobservatory/ooniprobe/domain/VersionCompareTest.java:32]: Not a very complex `build` is it? 😛

**Review by sdsantos — APPROVED** [review]

File: app/src/test/java/org/openobservatory/ooniprobe/domain/VersionCompareTest.java (hunk 3 above)
Comment (Filmaluco) [reply app/src/test/java/org/openobservatory/ooniprobe/domain/VersionCompareTest.java:32]: allows for a centralized place to init the testing class. If for some reason the class changes, theres no need to change all tests, only the build method :D

File: app/src/test/java/org/openobservatory/ooniprobe/domain/GetTestSuiteTest.java (hunk 2 above)
Comment (Filmaluco) [reply app/src/test/java/org/openobservatory/ooniprobe/domain/GetTestSuiteTest.java:42]: won't that test all the elements of the array as well...?

**Pull request merged**